prow/plugins/approve/approvers/owners.go

```diff
@@ -591,8 +593,9 @@ The full list of commands accepted by this bot can be found [here](https://go.k8
 Needs approval from an approver in each of these OWNERS Files:
 
 {{range .ap.GetFiles .org .project}}{{.}}{{end}}
-You can indicate your approval by writing `+"`/approve`"+` in a comment
-You can cancel your approval by writing `+"`/approve cancel`"+` in a comment
+OWNER approvers can indicate their approval by writing `+"`/approve`"+` in a comment
+OWNER approvers can cancel approval by writing `+"`/approve cancel`"+` in a comment
+A PR needs both /approve and /lgtm to be merged by the bot. An OWNER approver's /lgtm automatically implies an /approve.
```

Comment:
Whether or not `/approve` and/or `/lgtm` are required to merge depends on how the merge is configured (in `mungegithub` and/or `tide`)

Reply:
I was under the impression that a PR needs the following labels
1. "lgtm"
2. "approve"
with the additional confusion thrown in that lgtm from an approver (from the owners file) is an automatic approve.

Reply:
Can you point me to the place where the merge is configured? I am happy to change the logic, but at this point it's unclear to me how the logic flows.

Reply:
In the most common configuration, that is true, but there is no hard requirement for that to be the case. Not sure how we can make this comment more clear as the configuration for the merge behavior is not available to this plugin.

Reply:
I see. Thank you for explaining. I'll go with your suggestion.